cub/cub/util_macro.cuh

```diff
@@ -107,16 +97,6 @@ constexpr _CCCL_HOST_DEVICE auto max CUB_PREVENT_MACRO_SUBSTITUTION(T&& t, U&& u
 #  define CUB_ROUND_DOWN_NEAREST(x, y) (((x) / (y)) * y)
 #endif
 
-#ifndef CUB_STATIC_ASSERT
-#  ifndef DOXYGEN_SHOULD_SKIP_THIS // Do not document
-#    define CUB_CAT_(a, b) a##b
-#    define CUB_CAT(a, b)  CUB_CAT_(a, b)
-#  endif // DOXYGEN_SHOULD_SKIP_THIS
-
-/// Static assert
-#  define CUB_STATIC_ASSERT(cond, msg) typedef int CUB_CAT(cub_static_assert, __LINE__)[(cond) ? 1 : -1]
-#endif
```

Comment:
Are those macro's part of the public API and need to go through a deprecation process before being removed?

Reply:
My vote goes to just drop it